test/test.bzl

```diff
@@ -0,0 +1,24 @@
+load("@npm_bazel_typescript//:defs.bzl", "ts_library")
+
+test_deps = [
+    "test/wtf_mock",
+    "test/test_fake_polyfill",
+    "lib/zone",
+    "lib/common/promise",
+    "test/main",
+]
+
+def get_web_test_deps():
```

Comment:
this isn't called anywhere yet?

We generally prefer to just expand this out into the BUILD file. It's a bit repetitive to have 5 ts_library targets with similar setup, however tooling like ts_auto_deps makes it manageable. By having this in a macro, tooling like buildozer will no longer be able to machine-edit the rules

Reply:
Got it, I will modify this later, because currently this settings not work... I will delete all `bazel` settings of test for now. So this PR is only about `npm_package`.